Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN npm run build
 
 FROM node:current-alpine AS production
 ENV NODE_ENV=production
+ENV NOTIFY_BASE_API_URL=https://api.notification.canada.ca
+ENV NOTIFY_API_KEY=""
+ENV NOTIFY_REPORT_A_PROBLEM_EMAIL=""
+ENV NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID=""
```

Comment:
Should we add this to the README or something?

Currently, if you just try and submit without and env vars, you get a runtime error.

![Uploading Screen Shot 2021-05-03 at 12.32.33.png…]()

Reply:
probably a good idea actually